src/Nethermind/Nethermind.Blockchain/IBlockTree.cs

```diff
@@ -167,6 +167,7 @@ public interface IBlockTree : IBlockFinder
         event EventHandler<BlockEventArgs> NewSuggestedBlock;
         event EventHandler<BlockReplacementEventArgs> BlockAddedToMain;
         event EventHandler<BlockEventArgs> NewHeadBlock;
+        event EventHandler<OnUpdateMainChainArgs> OnUpdateMainChain;
```

Comment:
What is now the difference between `BlockAddedToMain` and `OnUpdateMainChain`? Would be good to document

Reply:
Done